### DISCUSSION TITLE
There is no contributors column on the repository home page

### DISCUSSION THREAD

**Discussion by BeastyZ** [community/community#75467] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I own this [repo](https://github.com/BeastyZ/LLM-Verified-Retrieval). I can see my contributions in the Insights, but there are no contributors on the homepage. I‘ve checked the email and it is associated with my GitHub account.

**Comment by Hsin-Hung** [comment]
I believe it only shows the contributors section if you have more than one contributor including yourself.

**Reply by BeastyZ** [reply]
Hi, I don't think so. In some GitHub repos, the only contributor to the repo is the author himself. So, it could be something else that's causing the problem.

**Reply by Hsin-Hung** [reply]
Can you show an example of the repo you mentioned?

**Reply by BeastyZ** [reply]
I'm sorry, I made a mistake. Thank you for your reminder and help.
